mock/service/controller.go

```diff
@@ -149,11 +183,14 @@ func (s *service) ValidateVolumeCapabilities(
 	req *csi.ValidateVolumeCapabilitiesRequest) (
 	*csi.ValidateVolumeCapabilitiesResponse, error) {
 
-	i, _ := s.findVolNoLock("id", req.VolumeId)
-	if i < 0 {
-		return nil, status.Error(codes.NotFound, req.VolumeId)
+	if len(req.GetVolumeId()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, req.GetVolumeId())
```

Comment:
Since Volume ID is empty, maybe it is better to replace `req.GetVolumeID()` with `Volume ID cannot be empty`.

Reply:
Done